### ISSUE TITLE
[Wave8: Ideal Nav Skeleton] Update the design of the `Workspaces` empty state page

### ISSUE THREAD

**Issue by trjExpensify** [Expensify/App#34232]
Coming from [here](https://expensify.slack.com/archives/C036QM0SLJK/p1704890830941179?thread_ts=1704705882.255709&cid=C036QM0SLJK). 

The empty state workspaces page will look like this when we implement the new navigation: 

<img width="1020" alt="image" src="https://github.com/Expensify/App/assets/16232057/16d0515b-9187-4f0d-bf06-f8a1c207fb0a">

It doesn't look the best, so we want to update it to look something more like this: 

<img width="1015" alt="image" src="https://github.com/Expensify/App/assets/16232057/fe80f260-7815-47dd-981b-2f9ab7e01134">

Finalising the plan in the thread with Shawn, and I'll assign this out after. Jakub, I wonder if we can get this one merged into the ideal nav branch or not? 

CC: @hayata-suenaga @mountiny @shawnborton @JmillsExpensify @kosmydel @adamgrzybowski

**Comment by shawnborton** [comment]
Sharing an early mock here: 
![image](https://github.com/Expensify/App/assets/2319350/2693ff09-3080-441c-8a8d-d7a0d0fca5c7)

I think we just need to decide which simple illustration to use, as we don't quite have one that matches the building icon we use for "Workspaces" in the LHN. I wonder if we want to use something different than a building for workspaces, perhaps a briefcase actually does make more sense?

**Comment by kosmydel** [comment]
Hey, sure! I think we can add it to our branch! 
@MaciejSWM was redesigning other screens, so he will take care of this issue.

**Comment by MaciejSWM** [comment]
<img width="1512" alt="image" src="https://github.com/Expensify/App/assets/85628901/52b7df6b-48d4-43cd-a8f8-88f76bb6fb11">

**Comment by shawnborton** [comment]
Nice! I think for this one, I prefer one-column so that the three features are just stacked vertically.

How much space is above the workspaces card and below the top page header area? I think we want just 12px of space there, and it looks to be more than that right now. 

Then on mobile, this should use a similar layout but the padding would drop from 32px down to 20px.

**Comment by MaciejSWM** [comment]
Posting final result here. But this will be part of our big `ideal-nav-merge` PR so probably this ticket can be closed soon.
![image](https://github.com/Expensify/App/assets/85628901/8729c38a-7a6b-4420-a73f-8df1cf779d5c)
![image](https://github.com/Expensify/App/assets/85628901/76ba7005-ce64-4d9a-98af-d56ec7e57552)
![image](https://github.com/Expensify/App/assets/85628901/3bb69185-6f78-4dd0-92fd-4ba498382c42)

**Comment by shawnborton** [comment]
And to clarify, is that card using a max-width of 600px? It feels a bit skinnier than that, so just checking.

**Comment by mountiny** [comment]
Slack [convo](https://expensify.slack.com/archives/C036QM0SLJK/p1705608322589469)

**Comment by MaciejSWM** [comment]
Good catch @shawnborton - they are limited to 600px but the horizontal margin is also being counted as part of the maxWidth. Looks like all the cards are affected by this problem, I'll fix it.

**Comment by trjExpensify** [comment]
@kosmydel did this make it into the feature branch yet?

**Comment by kosmydel** [comment]
> @kosmydel did this make it into the feature branch yet?

Yes, this one is on the feature branch.

**Comment by trjExpensify** [comment]
Great, let's close this out then!
